### ISSUE TITLE
Prevent accidental navigation on public form page

### ISSUE THREAD

**Issue by wanlingt** [opengovsg/FormSG#4793]
**Is your feature request related to a problem? Please describe.**
If a user on the public form page accidentally navigates back or forwards on their browser (e.g. by swiping the trackpad) and returns to the public form page, they lose all information that they've already filled in

**Describe the solution you'd like**
Possible solution: A confirmation modal pops up to ask if they're sure they want to navigate away from the page

**Comment by amitogp** [comment]
Another potential solution could be save as draft. That will be prioritized later.

**Comment by amitogp** [comment]
This issue should not be there now because we have implemented back button and breadcrumbs.

**Comment by justynoh** [comment]
@amitogp i don't think this actually done, since this is referring to public form respondents navigating away from the public form page

**Comment by amitogp** [comment]
Aahh got it.. it's for respondents. Reopening now.

**Comment by tutntut** [comment]
closing for now as fixed by save draft
